package.json

```diff
@@ -71,6 +71,7 @@
     "postcss-loader": "^3.0.0",
     "prettier-eslint": "^9.0.1",
     "prettier-eslint-cli": "^5.0.0",
+    "property-expr": "^2.0.3",
```

Comment:
It would be a good idea to upgrade this dependency through `yup`, which uses this package instead of pinning it.
I checked a bit and it looks like `yup(0.30.0)` https://github.com/jquense/yup/blob/v0.30.0/package.json supports `property-expr(2.0.4)`. We might need to upgrade tests just in case.